curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/using-objects-for-lookups.md

```diff
@@ -14,25 +14,23 @@ Objects can be thought of as a key/value storage, like a dictionary. If you have
 Here is an example of a simple reverse alphabet lookup:
```

Comment:
Need a verbage update here too

Reply:
will this work `Here is an example of a simple reverse article lookup:` ?

Reply:
No "reverse".
 
> Here is an example of an article object: